module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -509,6 +520,27 @@ public function init()
         $this->sessionCache = $factory($namespace);
     }
 
+    /**
+     * Establish INN-Reach database connection
+     *
+     * @return resource|void
+     */
+    protected function getInnReachDb()
+    {
+        try {
+            if (!isset($this->innReachDb)) {
```

Comment:
When initialized to null above you can just compare with null here: `if (null === $this->innReachDb) {`. 
I'd also check this outside of the try-catch block, since exception handling always adds a bit of overhead.